torch/distributed/rpc/api.py

```diff
@@ -201,6 +202,9 @@ def shutdown(graceful=True):
     ``graceful=False``, this is a local shutdown, and it does not wait for other
     RPC processes to reach this method.
 
+    .. warning::
```

Comment:
question: is there an actual misusage for that? I would imagine user calls shutdown and will not use any RPC/Async related APIs if they read the shutdown doc carefully.

Reply:
Yeah, I don't think users could be so careless and do this. This might not be a very useful caveat.